### ISSUE TITLE
UI service

### ISSUE THREAD

**Issue by maybe-hello-world** [netunicorn/netunicorn#120]
We need a separate UI service that would provide:
- Admin view for people with sudo privileges. Overview of active experiments, active compilations, locked nodes, and all experiments for the past X days. Ability to cancel any active experiment. Possibly, later, the ability to look at the source code of any of the experiments.
- User view for everyone. Info about their active experiments and nodes they locked, and their last (not deleted) experiments. Ability to cancel their experiment.

Basically, the difference between sudo and user UI is what experiments they have access to (their own or all).

**Comment by maybe-hello-world** [comment]
Implemented
